wgengine/userspace.go

```diff
@@ -67,6 +67,7 @@ func NewUserspaceEngine(logf logger.Logf, tunname string, listenPort uint16, der
 
 	tuntap, err := tun.CreateTUN(tunname, device.DefaultMTU)
 	if err != nil {
+		// TODO: why log.Printf here and below instead of logf?
```

Comment:
No good reason. Should be logf (or no log at all).

Reply:
Changed to logf for now.